CHANGELOG.rst

```diff
@@ -17,6 +17,10 @@ Unreleased
 ----------
 * nothing unreleased
 
+[4.19.8]
+* feat: updated exception handling in serializer
+* feat: added is_in_provisioning_admin_group permission class
```

Comment:
Please update this to something which is more precise and do not use feat.

Reply:
@zamanafzal i have updated description in changelog file and used `chore` instead, kindly check.